### ISSUE TITLE
rgw/sfs: remove Traefik and implement default ingress

### ISSUE THREAD

**Issue by jhmarina** [s3gw-tech/s3gw#389]
### What needs to be done

* Remove Traefik as default ingress for `s3gw`
* Ensure that the default Rancher Manager Ingress controller is used (we believe it's `nginx`)
* Test the new configuration to ensure that it meets our requirements

### Why it needs to be done

The goal is to serve requests to s3gw using the ingress provided by the Rancher Manager, instead of relying on Traefik.

### Acceptance Criteria

- Traefik has been successfully removed from `s3gw`
- The default Rancher Manager Ingress controller has been configured to handle traffic for our application.
- The new configuration has been tested thoroughly to ensure that it meets our requirements.

### Additional Information

This issue depends on #388

**Comment by jhmarina** [comment]
I'm putting this issue on hold until we sort out [it's dependency](https://github.com/aquarist-labs/s3gw/issues/388)

**Comment by jhmarina** [comment]
Moving it to the backlog. The UI backend is still the dependency.

**Comment by giubacc** [comment]
@jhmarina just for confirmation, are we still blocked with this right?

**Comment by jhmarina** [comment]
@giubacc I believe we are until the UI backend is further developed, right @jecluis?

**Comment by jecluis** [comment]
That's the idea, yes.

**Comment by jecluis** [comment]
Related to #720

**Comment by giubacc** [comment]
This topic seems quite outdated now in relation to the LH integration.
What should we do with this @jecluis ?

**Comment by jecluis** [comment]
This is still very much needed now that we no longer require CORS for the UI to work.
